rita/src/main.rs

```diff
@@ -77,37 +77,42 @@ fn main() {
 
     trace!("Starting with Identity: {:?}", my_ident);
 
-    let (tx, rx) = mpsc::channel();
 
-    let tx1 = mpsc::Sender::clone(&tx);
+    let (debt_keeper_input, debt_keeper_output) = DebtKeeper::start(Int256::from(1), Int256::from(-100000000000000i64), Int256::from(100), 2u32);
+
+    let payment_controller_input = PaymentController::start(
+        &my_ident,
+        Arc::new(Mutex::new(debt_keeper_input.clone()))
+    );
+
+    let payment_controller_input1 = payment_controller_input.clone();
```

Comment:
Why is this variable numbered? If it's got a distinct purpose why doesn't the name express it?

Reply:
It's a clone of a channel input. I would be curious what naming scheme you'd prefer. We could name it after what it was intended to be used by, but that might get very long winded.

Reply:
I understand now, so there's two different components chatting with the payment controller? And shadowing the same variable name would get the borrow checke angry?

Reply:
(or simply overwriting the variable)

Reply:
@jtremback

Reply:
I changed it so there is a master, which we clone, and just pass in variables which we just overwrite each time